back/pkg/utils/distinct.go

```diff
@@ -0,0 +1,20 @@
+package utils
```

Comment:
es gibt in entgo einen sql distinct selector `sql.Distinct` oder so... aber ich hab noch nicht rausgefunden wie man den in die query bekommt

Reply:
ich habs auch nicht hinbekommen :D